app/assets/stylesheets/components/_main_layout.sass

```diff
@@ -0,0 +1,32 @@
+$Main-background: $Theme-white
+$Main-color: $Theme-blue
+$Main-height-header: $Theme-height-header
+$Main-height-footer: $Theme-height-footer
+
+.MainLayout
+  display: flex
+  flex-direction: column
+  height: 100%
+  text-align: center
+
+.MainLayout-header
+  flex-shrink: 0
+  height: $Main-height-header
+  background: $Main-background
+  color: $Main-color
```

Comment:
Think about this component's name. Do you believe it should be giving any other style other than layout?